src/test/java/org/minions/devfund/sergio/SquarelotronTest.java

```diff
@@ -0,0 +1,347 @@
+package org.minions.devfund.sergio;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test methods for Squarelotron.
+ *
+ * @author Sergio Navarro
+ * @since 4/20/2018
+ */
+public class SquarelotronTest {
+
+    private static Squarelotron squarelotron;
+
+    /**
+     * Tests the values in the square of 2x2.
+     */
+    @Test
+    public void testSetValuesForSize2x2() {
+        final int size = 2;
+        final int[][] expected = {{1, 2}, {3, 4}};
+        squarelotron = new Squarelotron(size);
+        assertEquals(Squarelotron.showSquare(expected),
+                Squarelotron.showSquare(squarelotron.getOriginalSquareArray()));
+    }
+
+
+    /**
+     * Tests the values in the square of 5x5.
+     */
+    @Test
+    public void testSetValuesForSize5x5() {
+        final int size = 5;
+        final int[][] expected = {{1, 2, 3, 4, 5}, {6, 7, 8, 9, 10}, {11, 12, 13, 14, 15},
+                {16, 17, 18, 19, 20}, {21, 22, 23, 24, 25}};
+        squarelotron = new Squarelotron(size);
+        assertEquals(Squarelotron.showSquare(expected),
+                Squarelotron.showSquare(squarelotron.getOriginalSquareArray()));
+    }
+
+    /**
+     * Tests the values in the square of 4x4.
+     */
+    @Test
+    public void testGetNumberOfRingsInA4x4Square() {
+        final int size = 4;
+        squarelotron = new Squarelotron(size);
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Make the enclosing method "static" or remove this set. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2696)